pkg/antctl/raw/multicluster/create/clusterclaim.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type clusterClaimOptions struct {
+	namespace  string
+	clusterID  string
+	clusterSet bool
+}
+
+var clusterClaimOpt *clusterClaimOptions
+
+var clusterClaimExamples = strings.Trim(`
+Create a ClusterClaim in a leader or member cluster
+$ antctl mc create clusterclaim <CLUSTER_CLAIM_NAME> -n <NAMESPACE> --cluster-id <CLUSTER_ID> --clusterset 
```

Comment:
Let us remove the ` --cluster-id` argument (see discussion here: https://github.com/antrea-io/antrea/pull/3474/files#r886325540)

And rename `<CLUSTER_CLAIM_NAME>` to `<cluster ID>`